drivers/hrldas/PedoTransferSR2006Mod.F90

```diff
@@ -162,6 +162,9 @@ subroutine PedoTransferSR2006(NoahmpIO, noahmp, Sand, Clay, Orgm)
                   + sr2006_psi_e_b*psi_et         &
                   + sr2006_psi_e_c
     
+    theta_33    = max(10.0**-5.0,theta_33)  ! For numerical stability
+    theta_1500  = max(10.0**-5.0,theta_1500)  ! For numerical stability
```

Comment:
this may cause another issue that log(theta_33) - log(theta_1500) = 0 and then bexp = xx/ 0, leading to floating issue. maybe set different min values for theta_33 and theta_1500 with  theta_1500 having a smaller min value to ensure log(theta_33) - log(theta_1500) > 0.

Reply:
Yes, you are right. I have updated. Thanks

Reply:
What is the typical range of theta_33? Would 1e-3 be too large for the min value of theta_33? Similarly for theta_1500. For example, do we need to make their min values smaller, e.g., min(theta_33) = 1e-5, and min(theta_1500) = 1e-6?

Reply:
theta_1500 is the wilting point, and theta_33 is the field capacity.

The general range for the wilting point is 0.01-0.3, and for field capacity, it is 0.05-0.4


1e-5 is very small, with almost no soil moisture condition. If think about the table value for different soil types, the minimum value has 0.006 wilting point and 0.10 field capacity, except for water.

Also, there is another constraint at the end of this code, that use 0.17 and 0.01 for field capacity and wilting point.

https://github.com/NCAR/noahmp/blob/1a853a5df5fc706af4b96cf27d4d41affaad57df/drivers/hrldas/PedoTransferSR2006Mod.F90#L187

https://github.com/NCAR/noahmp/blob/1a853a5df5fc706af4b96cf27d4d41affaad57df/drivers/hrldas/PedoTransferSR2006Mod.F90#L188

Reply:
Thank you for the clarification. This makes sense to me. Your current values look good then.